src/app/core/entity-components/entity-details/form-entity-select/form-entity-select.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, Input } from "@angular/core";
+import { ENTITY_MAP } from "../entity-details.component";
+import { Entity, EntityConstructor } from "../../../entity/entity";
+import { DynamicComponentConfig } from "../../../view/dynamic-components/dynamic-component-config.interface";
+import { isRegisteredDynamicComponent } from "../../../view/dynamic-components-map";
+import { AbstractControl } from "@angular/forms";
+
+/**
+ * A view that shows an entity-select
+ * optimized so that it can be used in a configurable forms-component
+ */
+@Component({
+  selector: "app-form-entity-select",
+  templateUrl: "./form-entity-select.component.html",
+})
+export class FormEntitySelectComponent<E extends Entity> {
```

Comment:
Can we merge this `FormEntitySelectComponent` with the `EntitySelectComponent` implementation? Apart from (only slightly) different inputs the functionality they offer seems to be the same and this one would be more convenient to use everything, I think.

Reply:
I chose not to, though I am not sure which way is better.
The reason I chose to make two different components instead of one is due to the single-responsibility principle. It felt wrong to add more inputs to the `EntitySelectComponent` since it's overloaded with quite a few already. The `FormEntitySelectComponents` only makes the inputs visible that are required when in a form; all other inputs should not be available (or should be added explicitly).

Reply:
I think the FormEntitySelectComponent is actually easier to use - the usual approach will be to have the Entity*Block as template rather than something custom, for example.  And having two implementations has the danger of getting confused which one to use where.

We can have a look at it together if it really bloats things up too much ... but this class doesn't seem to contain too much logic. Maybe the selection getter could even fit into the FormComponent template.